Packs/Cryptocurrency/Integrations/Cryptocurrency/Cryptocurrency.py

```diff
@@ -0,0 +1,86 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+import urllib3
+import traceback
+from typing import Dict, List
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+BITCOIN = 'bitcoin'
+INTEGRATION_NAME = 'Cryptocurrency'
+SCORE = {
+    4: 'Critical',
+    3: 'Bad',
+    2: 'Suspicious',
+    1: 'Good',
+    0.5: 'Informational',
+    0: 'Unknown'
+}
+
+
+def get_bitcoin_reputation(addresses) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    score = 2
+    for address in addresses:
+        dbot_score = Common.DBotScore(
+            indicator=address,
+            indicator_type=DBotScoreType.CRYPTOCURRENCY,
+            integration_name=INTEGRATION_NAME,  # Vendor
+            score=score  # Suspicious
+        )
+        crypto_context = Common.Cryptocurrency(
+            address=address,
+            address_type=BITCOIN,
+            dbot_score=dbot_score
+        )
+        hr = f'Cryptocurrency reputation for address {address} was set to {SCORE[score]}'
```

Comment:
Let's change the HR to a table with the data, similar to the context output.
Address
AdddressType
Reputation (DBotScore)

Reply:
<img width="1619" alt="Screen Shot 2020-12-07 at 14 01 37" src="https://user-images.githubusercontent.com/21142167/101348694-bea91780-3894-11eb-8ce5-9ad3a5aa1762.png">